DQMOffline/Trigger/python/WprimeMonitor_cff.py

```diff
@@ -0,0 +1,93 @@
+import FWCore.ParameterSet.Config as cms
+
+from DQMOffline.Trigger.WprimeMonitor_cfi import hltWprimemonitoring
+
+WprimeEle115 = hltWprimemonitoring.clone()
+WprimeEle115.FolderName = cms.string('HLT/Wprime/WprimeEle115')
+WprimeEle115.nmuons = cms.uint32(0)
+WprimeEle115.nelectrons = cms.uint32(1)
+WprimeEle115.njets = cms.uint32(0)
+WprimeEle115.eleSelection = cms.string('pt>50 & abs(eta)<2.1 & (dr03TkSumPt+dr04EcalRecHitSumEt+dr04HcalTowerSumEt)/pt<0.1')
```

Comment:
can you please, drop the pT and eta cuts ?
how could you monitor the behavior of your path ?